app/jobs/email_missing_addresses_csv_job.rb

```diff
@@ -0,0 +1,32 @@
+class EmailMissingAddressesCsvJob < ApplicationJob
+  queue_as :default
+
+  BYTE_ORDER_MARK = "\uFEFF".freeze # Required to ensure Excel always reads CSV as UTF-8
+  EXPIRATION_TIME = 72.hours.to_i
```

Comment:
Is this enough? This sets the expiration of the s3 objects right?

Reply:
I think not the object but the url. Yeah, maybe, this was my first instinct. What do you think would be appropriate time?

Reply:
I'm imagining some users might take on the scale of a week to get back to us on this, is there any disadvantage to setting the expiration time to be much much longer?

Reply:
I guess these links do have user info so we don't want them to be available forever but a month would probably be alright and give users time to go back and avoid us having to resend links if users take multiple attempts to get the data right for example